packages/sourcing/test/fixtures/index.ts

```diff
@@ -0,0 +1,8 @@
+
+import { InvalidLocation, FileNotFound } from '../../src';
```

Comment:
In the new setup, we only re-export fixtures from the module. The tests will import the test subject from the `src` folder. This way we have `clean` fixture modules.